README.md

```diff
@@ -1,18 +1,109 @@
 # libgc
 
-libgc is a garbage collector for Rust. It works by providing a garbage-collected
-`Gc<T>` smart pointer in the style of `Rc<T>`.
+[![Bors enabled](https://bors.tech/images/badge_small.svg)](https://bors.tech)
 
-# Structure
+_libgc_ is a garbage collection library for Rust. It can be used as a standalone
+library, but it is highly recommended that programs are compiled with the
+companion [rustc fork](https://github.com/softdevteam/rustgc), which offers
+language support for much better performance.
+
+_libgc_ is in active development - there will be bugs!
+
+## Example
+
+_libgc_ provides a smart pointer, `Gc<T>`, which can be used to make
+garbage-collected values.
```

Comment:
Maybe add "An example, with the necessary setting up the global allocator, looks as follows:" or similar, just so that people realise they have to copy the weird looking stuff at the top of the example?

Reply:
Good point.

Reply:
Fixed: 2c3f464bab1f1dab1d543e2618347dbfced84e4e

Reply:
I mean: 6ab936a244ed6bbcbe3e0fc5841e1161d4162bd3 *